bcgov/HOUS-permit-portal
# Restrict registration/login to keycloak omniauth

## Description

- allow user to register with bceid
- remove database user authentication

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

Implements [BPHH-1266](https://hous-bssb.atlassian.net/browse/BPHH-1255)
Implements [BPHH-1263](https://hous-bssb.atlassian.net/browse/BPHH-1263)

## Steps to QA

1. Open the app and go to login. Select BCeID as the only option. 
2. On the BCeID login page, login with a dev BCeID (or go create one from the login page). 
3. Once logged in, you should be redirected back to the app and prompted to complete registration.
4. Once the registration form is submitted, a confirmation email is sent to verify the email
5. After email verification, the user can access the app.

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

n/a
